Xamarin.Forms.Core/Binding.cs

```diff
@@ -36,6 +36,7 @@ public Binding(string path, BindingMode mode = BindingMode.Default, IValueConver
 			Mode = mode;
 			StringFormat = stringFormat;
 			Source = source;
+			NullValue = nullValue;
```

Comment:
should be done in base ctor, as you defined `NullValue` as a property of `BindingBase`